client/.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# GraphQL codegen
+**/generated.ts
```

Comment:
I'll add a README explaining what this is in a later commit. 

This is something automatically generated during development and does not need to be checked into git (in order to reduce the likelihood of merge conflicts).